src/components/detail/Detail.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { getCharacter } from '../../services/get-character';
+import PropTypes from 'prop-types';
+export default class Detail extends Component {
+state={}
+componentDidMount(){
```

Comment:
I'd add a line of whitespace after state and your component did mount just to make the separate pieces of your component a little more readable